translations/admin_ext.ca.yaml

```diff
@@ -432,7 +432,7 @@ conversionOffset: Conversion Offset
 default_unit: Default Unit
 min_max_times: Min / Max Times
 reset: Reset
-password_hint: Password must contain at least 10 characters!
+password_hint: Password does not meet the criteria
```

Comment:
@kevinloeber-twocream The translations of the non-english files are still there. Could you please have another look? Thanks

Reply:
Hey @robertSt7 i removed all the translation files, but github says that the `admin_ext.nl.yml` file is empty, but it is not.